### PR TITLE
Preliminary work for 1849

### PR DESCRIPTION
**Pull request by tysen**
Adds a barebones 1849 configuration as a PoC for the Tile/Path/Track updates
to support other gauges.

### CODE REVIEW

File: lib/engine/share_price.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ class SharePrice
       'z' => :par_2,
       'C' => :convert_range,
       'm' => :max_price,
+      'u' => :phase_limited,
```
Comment (jenf) [inline lib/engine/share_price.rb:23]: need to merge up for this

Comment (tysen) [reply lib/engine/share_price.rb:23]: this is up to date

File: assets/app/view/game/part/track.rb [hunk 2]
```diff
@@ -15,6 +15,21 @@ class Track < Snabberb::Component
           color: '#000000',
           width: 8,
           dash: '0',
+          broad: {
```
Comment (jenf) [inline assets/app/view/game/part/track.rb:18]: How does this interact with user settings?

Comment (tysen) [reply assets/app/view/game/part/track.rb:18]: User settings can override this.

Comment (jenf) [reply assets/app/view/game/part/track.rb:18]: It's probably something to fix as part of a separate PR to allow narrow and dual to be overriden individually.

File: lib/engine/game/g_1849.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+      EBUY_OTHER_VALUE = false # allow ebuying other corp trains for up to face
+      HOME_TOKEN_TIMING = :float
+      SELL_AFTER = :operate
+      SELL_BUY_ORDER = :sell_buy
+
+      MARKET_TEXT = Base::MARKET_TEXT.merge(phase_limited: 'Can only enter during phase 16').freeze
+
+      STOCKMARKET_COLORS = {
+        par: :yellow,
```
Comment (jenf) [inline lib/engine/game/g_1849.rb:36]: Use Ruby 1.9 syntax here par => :yellow

**Comment by jenf** [comment]
Apart from the minor style change, this looks good. Adding narrow gauge will need Toby's viewpoint.

**Comment by jenf** [comment]
Looks good, the custom style should be done separately. Failure important change so @tobymao should review

**Review by tobymao — APPROVED** [review]

**Pull request merged**